docs/information/supported_adapters.md

```diff
@@ -130,9 +130,9 @@ Zigbee2MQTT officially supports the following adapters:
     <td><a href="https://www.tindie.com/products/GiovanniCas/zigbee-hat-with-cc2538-for-raspberry/">Tindie</a></td>
   </tr>
   <tr>
-    <td><img src="../images/cc2652p-z-bee-duo.jpg"></td>
```

Comment:
Please remove the old picture from this repo

Reply:
Ok, done